Pod/Classes/SEGLeanplumIntegration.h

```diff
@@ -7,7 +7,12 @@
 
 #import <Analytics/SEGAnalytics.h>
 #import <Foundation/Foundation.h>
-#import <Leanplum/Leanplum.h>
+
+#if TARGET_OS_IOS
```

Comment:
How does this env var get set? Is the user expected to set this? If so should we add this to the readme?